app/src/main/java/com/saotome/imed_app/ui/fragments/EditDrugFragment.kt

```diff
@@ -5,7 +5,9 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
 import com.saotome.imed_app.R
+import com.saotome.imed_app.databinding.FragmentEditDrugBinding
 
 // TODO: Rename parameter arguments, choose names that match
```

Comment:
Do you still need this TODO comment?

Reply:
No, I don't. Removed.
670270c3977d061440ddc4f91f242f72151084ed